database/DevHome.Database/DevHomeDatabaseContext.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using DevHome.Database.DatabaseModels.RepositoryManagement;
+using Microsoft.EntityFrameworkCore;
+
+namespace DevHome.Database;
+
+public class DevHomeDatabaseContext : DbContext
+{
+    public DbSet<Repository> Repositories { get; set; }
+
+    public DbSet<RepositoryMetadata> RepositoryMetadatas { get; set; }
+
+    public string DbPath { get; }
+
+    public DevHomeDatabaseContext()
+    {
+        // Not the final path.  It will change before going into main.
+        var folder = Environment.SpecialFolder.LocalApplicationData;
```

Comment:
Db location should be in the LocalFolder of Windows.ApplicationData.Current where it is safely tied to the lifetime of the Dev Home package and will be safe from collisions with other flavors of Dev Home.

Reply:
Correct.  Yes.  I do plan to move this to the application data folder.  This is a temp location and will be changed.